src/components/metricsConfiguration/TraceabilityConfig.Component.tsx

```diff
@@ -0,0 +1,187 @@
+import { useState } from "react";
+import { Typography, styled, Box, Button, Stack } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import { useAppDispatch } from "state/store";
+import { TRACEABILITY_COFFEE_ARR, TRACEABILITY_PALM_ARR } from "constants/settings.constants";
+import { Formik, Form, Field } from "formik";
+import { Checkbox } from "formik-mui";
+import ModalContainer, { ButtonData } from "components/common/Modal.component";
+import { traceabilityMetricsActionCreator } from "state/actions/settings/config-lead/configLead.action";
+import BackArrow from "assets/images/backArrow.svg";
+import greenTickIcon from "assets/images/greenTick-icon.svg";
+import { ConfigFormI, TraceabilityMetricsI } from "model/settings.model";
+
+const ConfigContainer = styled(Box)(({ theme }) => ({
+  padding: "30px",
+  width: "100%",
+  "& h1": {
+    padding: "12px 0",
+  },
+  "& .MuiTypography-body1": {
+    color: theme.palette.grey[700],
+    paddingBottom: "15px",
+  },
+  "& .backButton": {
+    color: theme.palette.common.black,
+    paddingLeft: "10px",
+  },
+}));
+
+const TraceabilityContainer = styled(Box)(({ theme }) => ({
+  background: theme.palette.common.white,
+  padding: "40px 50px 15px 50px",
+  height: "480px",
+  "& form": {
+    display: "flex",
+    flexDirection: "column",
+    flexWrap: "wrap",
+    height: "100%",
+    fontFamily: "Roboto Regular",
+    "& label": {
+      marginBottom: "25px",
+      "& .MuiCheckbox-root": {
+        marginRight: "20px",
+        padding: "0",
+        width: "13px",
+        height: "10px",
+      },
+    },
+  },
+}));
+
+const ButtonContainer = styled(Box)(({ theme }) => ({
+  background: theme.palette.common.white,
+  position: "fixed",
+  bottom: 0,
+  width: "100%",
+  right: 0,
+  padding: "30px",
+  boxShadow: "0px -2px 4px rgb(0 0 0 / 10%)",
+  display: "flex",
+  justifyContent: "flex-end",
+  "& button": {
+    padding: "4px 50px",
+  },
+}));
+
+interface TraceabilityFormI {
+  configuredData: ConfigFormI;
+}
+
+const ConfirmationModalBody = () => {
+  return (
+    <Stack sx={{ paddingLeft: "30px" }}>
+      <Typography variant='body2' className='message-description'>
+        You have completed the <br />
+        <b>Traceability Metrics Configuration. </b>
+      </Typography>
+      <Typography variant='body2' sx={{ paddingTop: "30px" }}>
+        Do you want to configure Sustainability Metrics now? You can configure them later as well.{" "}
```

Comment:
{" "} Do you need this symbol?
```suggestion
        Do you want to configure Sustainability Metrics now? You can configure them later as well.{" "}
```